src/BrazeKit-dev.js

```diff
@@ -129,6 +129,8 @@ var constructor = function () {
             quantity,
             eventAttributes
         );
+
+        return reportEvent;
```

Comment:
wouldn't this be `return reportEvent === true`? thats how at least other log events functions have it as and I think it makes sense in case its not `true`? but also the video you shared for testing seems to be working as expected

Reply:
Good attention to detail!  Yes so it seems that Braze returns `true` from when you call their login method already, so, so doing just `return reportEvent` is fine.  I'm not sure why we do return `reportEvent === true` since it is essentially the same thing.  For now, to be consistent, I will just follow that format even though it's unnecessary, but I can make a ticket to refactor in the future to just return the boolean instead of the equivalence.